Android/app/src/main/res/values-es/strings.xml

```diff
@@ -88,6 +88,7 @@
     <!--Daily Report-->
     <string name="daily_report_subtitle">Reporte Diario</string>
     <string name="daily_report_product_popularity">Popularidad del producto</string>
+    <string name="daily_report_progress_message">Cargando gráfico</string>
```

Comment:
I'm using google translate for Spanish, is that okay?

Reply:
Yeah absolutely - that's what I used for all the translations :grin: